subsys/bluetooth/host/att.c

```diff
@@ -2457,6 +2457,9 @@ static uint8_t att_error_rsp(struct bt_att_chan *chan, struct net_buf *buf)
 #if defined(CONFIG_BT_ATT_RETRY_ON_SEC_ERR)
 	/* Check if error can be handled by elevating security. */
 	if (!att_change_security(chan->chan.chan.conn, err)) {
+		/* Cancel ongoing timeout work */
```

Comment:
looks good but I'd add extra comment that normally this is cancelled in att_handle_rsp() but this is special case

Reply:
done